pkg/scalers/kafka_scaler.go

```diff
@@ -183,7 +183,7 @@ func parseKafkaMetadata(config *ScalerConfig) (kafkaMetadata, error) {
 			meta.cert = config.AuthParams["cert"]
 			meta.key = config.AuthParams["key"]
 			meta.enableTLS = true
-		} else {
+		} else if val != "disable" {
```

Comment:
What happens if the value is not `enable` nor `disable`?

Reply:
Oh I see now.

Reply:
Could it be better if we remove the whole else block?

Reply:
onle `enabled` will enable it and other values will do nothing

Reply:
Using `disabled` to be explicit in the scaled object should not be blocked though. Which is the problem today.

Better be explicit than make assumptions.